student/models.py

```diff
@@ -1,3 +1,30 @@
-# from django.db import models
+from django.db import models
+from django.conf import settings
 
-# Create your models here.
+
+class Gender(models.TextChoices):
+    MALE = "M", "Male"
+    FEMALE = "F", "Female"
+    UNSPECIFIED = "U", "Unspecified"
+
+
+class EducationalInstitution(models.Model):
+    institution_name = models.CharField(max_length=255)
```

Comment:
Are you missing a link field here back to the students model?

Reply:
@ifireball 
I thought if i used a many to many field I don't need an additional field here because I can use the related_name of the field in Student.. should I still add a field?

Reply:
You typically need link fields in both directions.

Reply:
Not seeing updates in the code here yet

Reply:
We changed the field in Student to a foreign key so I don't think a link back is required?

Reply:
its useful for all kinds of links - suppose you have an institution object in the code - how do you map it back to the students in that institution?